lib/alpha.js

```diff
@@ -87,9 +87,9 @@ for (var _locale, _i = 0; _i < arabicLocales.length; _i++) {
 } // Source: https://en.wikipedia.org/wiki/Decimal_mark
 
 
-var dotDecimal = ['ar-EG', 'ar-LB', 'ar-LY'];
+var dotDecimal = ['ar-EG', 'ar-LB', 'ar-LY', 'en-AU', 'en-CA', 'zh-CN', 'hr', 'es-DO', 'ar-EG', 'es-SV', 'es-GT', 'es-HN', 'pa-IN', 'en-IE', 'en-JM', 'ja', 'ar-JO', 'ko-KP', 'ko-KR', 'ar-LY', 'ar-LB', 'de-LI', 'de-LU', 'es-MX', 'ne', 'en-NZ', 'es-NI', 'pa-PK', 'es-PA', 'en-PH', 'es-PR', 'si', 'en-ZA', 'it-CH', 'de-CH', 'fr-CH', 'zh-TW', 'en-GB', 'en-US', 'en-ZW'];
```

Comment:
Change this code from `src/lib` instead, once you run the test, this file will be automatically generated.

Could you also add some test cases for this? At least for a few of the classes that use comma vs. dot.

Reply:
oh yes, didn't recognize.